vital/bindings/python/vital/types/descriptor_set.cxx

```diff
@@ -30,17 +30,41 @@
 
 #include <pybind11/stl.h>
 
-#include "descriptor_class.cxx"
+#include <vital/types/descriptor_set.h>
 
 namespace py = pybind11;
 
+typedef kwiver::vital::descriptor_set desc_set;
+typedef kwiver::vital::simple_descriptor_set s_desc_set;
+
+std::shared_ptr<s_desc_set>
+new_desc_set()
+{
+  return std::shared_ptr<s_desc_set>(new s_desc_set());
+}
+
+std::shared_ptr<s_desc_set>
+new_desc_set1(py::list py_list)
+{
+  std::vector<std::shared_ptr<kwiver::vital::descriptor>> desc_list;
+  for(auto py_desc : py_list)
+  {
+    desc_list.push_back(py::cast<std::shared_ptr<kwiver::vital::descriptor>>(py_desc));
+  }
+  return std::shared_ptr<s_desc_set>(new s_desc_set(desc_list));
+}
```

Comment:
Would std::make_shared<>() be better here?

Reply:
no significant preference, I'll make the change since it shortens code